collections/_testimonials/24-LuXDEM.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Evaluation of erosion phenomena in a nozzle for Abrasive Water Jet Cutting"
+author: "Prasad ADHAV"
+author_link: "https://luxdem.uni.lu/team/padhav/"
+organisation: "LuXDEM, University of Luxembourg, LU"
+organisation_link: ""
+img: testimonial-24-LuXDEM.png
```

Comment:
Needs to be the same name as your picture below. Could you please rename the picture?

Reply:
Pushed the changed.